tracex_project/extraction/templates/result.html

```diff
@@ -27,23 +27,23 @@ <h1>Success! The TracEX pipeline produced the following results:</h1>
         });
     </script>
 
-    {% if xes_html %}
+    {% if trace_table %}
         <button id="toggleButton1">Show/Hide extracted Event Log from Journey</button>
         <div id="contentWrapper1" style="display: none;">
-            <p><b>This is the event log extracted from the journey above (single trace)</b></p>
-            <div id="xesContent1" class="xes_container">{{ xes_html|safe }}</div>
+            <p><b>This is the event log extracted from the journey above.</b></p>
```

Comment:
"This is the event log extracted from the journey above." => I think it should be "Trace" instead of event log